kaldb/src/test/java/com/slack/kaldb/metadata/service/ServicePartitionMetadataTest.java

```diff
@@ -1,18 +1,61 @@
 package com.slack.kaldb.metadata.service;
 
+import static com.slack.kaldb.metadata.service.ServicePartitionMetadata.findPartitionsToQuery;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.awaitility.Awaitility.await;
 
+import brave.Tracing;
+import com.slack.kaldb.metadata.zookeeper.MetadataStore;
+import com.slack.kaldb.metadata.zookeeper.ZookeeperMetadataStoreImpl;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import org.apache.curator.test.TestingServer;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 public class ServicePartitionMetadataTest {
 
+  private SimpleMeterRegistry metricsRegistry;
+  private MetadataStore zkMetadataStore;
+  private ServiceMetadataStore serviceMetadataStore;
+  private TestingServer testZKServer;
+
+  @Before
+  public void setUp() throws Exception {
+    Tracing.newBuilder().build();
+
+    metricsRegistry = new SimpleMeterRegistry();
+    testZKServer = new TestingServer();
+
+    // Metadata store
+    com.slack.kaldb.proto.config.KaldbConfigs.ZookeeperConfig zkConfig =
+        com.slack.kaldb.proto.config.KaldbConfigs.ZookeeperConfig.newBuilder()
+            .setZkConnectString(testZKServer.getConnectString())
+            .setZkPathPrefix("indexerTest")
```

Comment:
Probably set the zk path to something like `servicePartitionMetadataTest` or similar, as this looks leftover from the original copy.